tests/polynomial/lower_add.mlir

```diff
@@ -41,3 +41,51 @@ func.func @test_lower_add_prime_cmod() -> !polynomial.polynomial<#ring_prime> {
   // CHECK: return  [[TRUNC_RESULT]] : [[T]]
   return %poly2 : !polynomial.polynomial<#ring_prime>
 }
+
+func.func @test_lower_add_tensor() -> tensor<2x!polynomial.polynomial<#ring>> {
```

Comment:
```suggestion
// CHECK-LABEL: @test_lower_add_tensor
func.func @test_lower_add_tensor() -> tensor<2x!polynomial.polynomial<#ring>> {
```

Though I didn't do it elsewhere in this file due to my past ignorance, each test should have a `CHECK-LABEL` to separate the CHECK directives that are meant for each test function. Otherwise (concurrently with some other bug) a CHECK from before line 44 could match ops in this test, and produce a confusing failure.

Feel free to add the other check-label directives for this file if you have a sec.

Reply:
Done (7ee24021b8285b549dd2fd07d8b2fa6beac56cbc)